clusterloader2/state/measurements.go

```diff
@@ -0,0 +1,26 @@
+package state
+
+import (
+	"fmt"
+)
+
+// MeasurementsState represents state of currently running measurements.
+type MeasurementsState map[string]bool
```

Comment:
Given that measurements can be started/stopped in parallel, I'm fairly convinced that you need synchronization here.

The same for all other stuff.

Reply:
Synchronization add.
Also namespaces state logic has been moved to namespaces.go